### PR TITLE
Build libsnowflakeclient with /MT option

### PR DESCRIPTION
**Pull request by howryu**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -166,6 +168,7 @@ endif()
 
 if (WIN32)
     # Windows
+	option(DYNAMIC_RUNTIME "Dynamic runtime" ON)
```
Comment (smtakeda) [inline CMakeLists.txt]: indentation.

File: CMakeLists.txt [hunk 2]
```diff
@@ -222,7 +225,17 @@ endif ()
...
+	else()
+		set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
+ 		set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
+	endif()
 endif ()
 
 add_subdirectory(examples)
+
+if (BUILD_TESTS)
 add_subdirectory(tests)
```
Comment (smtakeda) [inline CMakeLists.txt]: Add the indentation.

**Comment by smtakeda** [comment]
@howryu added the tag `v0.1.2`

**Comment by howryu** [comment]
Thanks!

**Pull request merged**